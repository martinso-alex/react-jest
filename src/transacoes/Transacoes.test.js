import React from "react";
import { render } from "@testing-library/react";

import Transacao from "./Transacao";

describe("Componente de transações", () => {
	it("snapshot do componente não deve ser alterado", () => {
    const transacoes = [
      {data:'08/09/2020', transacao:'saque', valor:'20.00'},
      {data:'09/09/2020', transacao:'saque', valor:'40.00'}
    ];

    const { container } = render(
			<Transacao transacoes={transacoes} />
		);

		expect(container.firstChild).toMatchSnapshot();
	});
});
