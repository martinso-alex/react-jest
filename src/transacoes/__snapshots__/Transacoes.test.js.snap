// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transações snapshot do componente não deve ser alterado 1`] = `
<div
  class="transacao-container"
>
  <p />
  <p />
  <p>
    R$ 
    0
  </p>
</div>
`;
